python/lsst/afw/table/arrays.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from ._arrays import *
```

Comment:
I think in theory we could get rid of the `*Lib.py` intermediary completely and just import all the symbols directly into the `__init__.py`. These files are so short and are acting like `__init__.py` files so I'm not entirely convinced they are anything to worry about.